index.src.html

```diff
@@ -505,7 +505,7 @@ <h2 id="referrer-policy-delivery">Referrer Policy Delivery</h2>
     </li>
     <li>
       Via a <code>referrerpolicy</code> content attribute on an <{a}>,
-      <{area}>, <{img}>, <{iframe}>, or <{link}> element.
+      <{area}>, <{img}>, <{iframe}>, <{link}> or <{script}> element.
```

Comment:
Prior to your change we had a comma after or, can you add it back here please?

Reply:
Ah! I had double checked this, but had to remake the changes as my IDE was removing a ton of white-space which would've made this harder to review. Fixed! Thanks @domfarolino.